README.md

```diff
@@ -12,30 +12,58 @@ npm install eslint-config-brightspace
 
 ## Usage
 
-Set the `extends` property in the `.eslintrc.json` file, replacing `<environment>` with the desired environment-specific config.
+Shared configurations can be directly exported or included as part of a custom configuration in the `eslint.config..js` file.
+```js
+export { nodeConfig as default } from './index.js';
+```
+
+```js
+import { nodeConfig } from 'eslint-config-brightspace';
+
+export default [
+	...nodeConfig,
+	// Custom configuration
+];
+```
+
+### Utils
+
+Since the `--ext` flag is deprecated, include extensions by using the `addExtenstion` function from `utils.js`.
+```js
+import { nodeConfig } from 'eslint-config-brightspace';
+
+export default addExtensions(nodeConfig, ['.js','.html']);
+```
 
-```json
-{
-  "extends": "brightspace/<environment>"
-}
```

Comment:
Maybe prefix this section with a "Different Configurations for Different Directories" heading?